pkg/agent/cluster/cluster.go

```diff
@@ -49,6 +49,10 @@ func CAChecksum() string {
 	return os.Getenv("CATTLE_CA_CHECKSUM")
 }
 
+func CAStrictVerify() bool {
+	return strings.ToLower(os.Getenv("STRICT_VERIFY")) == "true"
```

Comment:
Would it make more sense to name this var `CATTLE_CA_STRICT_VERIFY` in line with our agent env vars?

Reply:
This is actually being called `STRICT_VERIFY` as it is being driven by the design doc. I've copied the environment variable name verbatim out of the doc.